tests.sh

```diff
@@ -422,8 +422,6 @@ function create_pool(){
     else
         pmsg "dmg pool create SUCCESS"
     fi
-
-    sleep 10
```

Comment:
Looks like we can remove another sleep from setup_pool() line number 438, too?

Reply:
Yeah, I think that's probably okay to remove, but I would like to run some sanity tests to make sure it isn't expected somehow